crates/ruff/resources/test/fixtures/flake8_type_checking/strict.py

```diff
@@ -49,7 +49,7 @@ def test(value: pkg.bar.A):
 
 
 def f():
-    # In un-strict mode, this _should_ rase an error, since `pkg` is used at runtime.
+    # In un-strict mode, this _should_ rase an error, since `pkg.bar` isn't used at runtime
```

Comment:
(These comments were just slightly misleading. The actual tests and results haven't changed.)